config/application.php

```diff
@@ -96,23 +99,9 @@
 /**
  * Custom Settings
  */
-Config::define('AUTOMATIC_UPDATER_DISABLED', true);
```

Comment:
We still want to disable automatic updates, don't we? Updates would come from the Bedrock upstream.